elated-egrets/bangazon-api
# Order/Product

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git pull ryDoubleProduct
git checkout ryDoubleProduct
python manage.py runserver
```

## Steps to test
Make sure you're in a virtual environment
Nuke migrations and database
In Bangazon project folder:
python manage.py makemigrations Api
python manage.py migrate
python manage.py runserver

Navigate to local/api/
See product is available on order
Add a user and category ID
Add a product to test adding a product with user and category ID
Navigate to local/api/order
See product information is available 
## What you changed
I have added the product information to the order form
## What was added to read me
I will update README after the check.
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
1. Has the readme been updated to reflect changes made?
1. Has a teammate approved the PR?

Then Ship It!
